Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+podTemplate(
+  cloud: 'openshift',
+  containers: [
+    containerTemplate(name: 'jnlp',
+      image: 'image-registry.openshift-image-registry.svc:5000/oasis/jenkins-agent-oasis:latest',
+      args: '${computer.jnlpmac} ${computer.name}',
+      envVars: [
+      secretEnvVar(key: 'OASIS_RHSM_USERNAME', secretName: 'oasis-rhsm', secretKey: 'username'),
+      secretEnvVar(key: 'OASIS_RHSM_PASSWORD', secretName: 'oasis-rhsm', secretKey: 'password'),
+      secretEnvVar(key: 'OASIS_RHSM_POOL_IDS', secretName: 'oasis-rhsm', secretKey: 'pool_ids'),
+      secretEnvVar(key: 'OASIS_RHSM_SERVER_HOSTNAME', secretName: 'oasis-rhsm', secretKey: 'hostname'),
+      secretEnvVar(key: 'OCP_PULL_SECRETS_OFFLINE_TOKEN', secretName: 'oasis-ci-pull-secrets', secretKey: 'offline_token')
+      ],
+      alwaysPullImage: true)
+]) {
+  node(POD_LABEL) {
+    def collectionDir = 'ansible_collections/oasis_roles/system'
```

Comment:
Since this is the only line that's not going to be identical between all the collections we produce, I'm hoping v2 of this looks REALLY simple!

Reply:
We know the repo name, so if we keep the "ansible_collection_<collectionname>" convention, even this literal gets to be dynamically generated.